src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -310,45 +310,47 @@ public sealed class SecurityIdentifier : IdentityReference, IComparable<Security
         //       the typecast
         //
 
+#if NETCOREAPP2_0
         private void CreateFromParts(IdentifierAuthority identifierAuthority, int[] subAuthorities)
+#else
+        private void CreateFromParts(IdentifierAuthority identifierAuthority, ReadOnlySpan<int> subAuthorities)
+#endif
         {
-            if (subAuthorities == null)
-            {
-                throw new ArgumentNullException(nameof(subAuthorities));
-            }
-
             //
             // Check the number of subauthorities passed in
             //
-
             if (subAuthorities.Length > MaxSubAuthorities)
             {
                 throw new ArgumentOutOfRangeException(
                     "subAuthorities.Length",
                     subAuthorities.Length,
-                    SR.Format(SR.IdentityReference_InvalidNumberOfSubauthorities, MaxSubAuthorities));
+                    SR.Format(SR.IdentityReference_InvalidNumberOfSubauthorities, MaxSubAuthorities)
+                );
             }
 
             //
             // Identifier authority is at most 6 bytes long
             //
 
-            if (identifierAuthority < 0 ||
-                (long)identifierAuthority > MaxIdentifierAuthority)
+            if (identifierAuthority < 0 || (long)identifierAuthority > MaxIdentifierAuthority)
             {
                 throw new ArgumentOutOfRangeException(
-nameof(identifierAuthority),
+                    nameof(identifierAuthority),
                     identifierAuthority,
-                    SR.IdentityReference_IdentifierAuthorityTooLarge);
+                    SR.IdentityReference_IdentifierAuthorityTooLarge
+                );
             }
 
             //
             // Create a local copy of the data passed in
             //
 
             _identifierAuthority = identifierAuthority;
-            _subAuthorities = new int[subAuthorities.Length];
-            subAuthorities.CopyTo(_subAuthorities, 0);
+#if NETCOREAPP2_0
+            _subAuthorities = = (int[])subAuthorities.Clone();
```

Comment:
Yep, double-equal.

Reply:
Yup, saw it as soon as I started looking this morning but I had to get back from work before I could fix it. Silly mistake, fixed.